testdata/pki/gocql.cnf

```diff
@@ -0,0 +1,16 @@
+[req]
+default_bits = 2048
+prompt = no
+default_md = sha256
+distinguished_name = dn
+
+[dn]
+CN = gocql
+
+[req_ext]
+basicConstraints = CA:FALSE
+keyUsage = digitalSignature, keyEncipherment
+subjectAltName = @alt_names
+
+[alt_names]
+URI = spiffe://test.cassandra.apache.org/cassandra-gocql-driver/integrationTest/gocql
```

Comment:
Note, I opted to include a URI:spiffe in the SAN of both the gocql and cassandra certificates as I intend to write a test that utilizes the mutual TLS feature introduced in 5.0.  I will create a follow on PR for this later.  I've already tried the feature out locally and it works great with gocql!